MAVProxy/mavproxy.py

```diff
@@ -936,8 +936,13 @@ def mkdir_p(dir):
 
 def log_writer():
     '''log writing thread'''
-    while not mpstate.status.exit:
-        mpstate.logfile_raw.write(bytearray(mpstate.logqueue_raw.get()))
+    while not mpstate.status.stop_event.is_set():
+        if not mpstate.logqueue_raw.empty():
```

Comment:
We still appear to be looking at the raw queue later in this function - conflict resolution error?

Reply:
The event is the exit event. This statement should read

```
while we are not supposed to exit:
   use the queue for logging operations
 ```
 
 Perhaps you missed the `not` when reading the code?  It's the same behavior as before. Run logging if we aren't exiting.